deploy/infra/deployment.yaml

```diff
@@ -490,8 +490,8 @@ spec:
             cpu: 100m
             memory: 256Mi
         volumeMounts:
-          - name: kubeconfig
-            mountPath: /etc/kubernetes/kubeconfig
+          - name: kubelet-config
+            mountPath: /var/lib/kubelet
```

Comment:
Why is this change required? The volume mount is only for within the pod. The actual volume that's still mounted is `/var/lib/kubelet` hostPath.

Reply:
I put the background in the PR description. The main reason is with TLS bootstrapping enabled, this kubeconfig will reference the client certificates in `/var/lib/kubelet/pki`. In this case, we should mount the whole `/var/lib/kubelet` into the pod.